content/blog/2017/2017-01-12-declarative-pipeline-beta-2.adoc

```diff
@@ -0,0 +1,122 @@
+---
+layout: post
+title: "Declarative Pipeline Syntax Beta 2 release"
+tags:
+- plugins
+- pipeline
+author: abayer
+---
+
+This week, we released the second beta of the new
+link:https://wiki.jenkins-ci.org/display/JENKINS/Pipeline+Model+Definition+Plugin[Declarative Pipeline syntax],
```

Comment:
I have been preferring links to plugins.jenkins.io whenever we reference plugins, mind updating?